samples/nrf_desktop/src/hw_interface/buttons.c

```diff
@@ -359,8 +375,7 @@ static void init_fn(void)
 		__ASSERT_NO_MSG(col[i].port < ARRAY_SIZE(port_map));
 
 		int err = gpio_pin_configure(gpio_devs[col[i].port],
-					     col[i].pin,
-					     GPIO_DIR_OUT);
+					     col[i].pin, GPIO_DIR_IN);
```

Comment:
If we only set column pin as output when it's needed, then we don't need to initially configure the pins. GPIO pins are Hi-Z (inputs) by default.

Actually this way of setting Hi-Z of output may harm readability. I think this for loop can be removed.

Reply:
Actually I prefer to have it in the known state in case this was configured by something (e.g. bootloader).